pkg/syncer/admissionhandler/pvcsi_admissionhandler.go

```diff
@@ -93,6 +93,10 @@ func (h *CSIGuestWebhook) Handle(ctx context.Context, req admission.Request) (re
 			admissionResp := validatePVC(ctx, &req.AdmissionRequest)
```

Comment:
can we enhance validatePVC and add check for `csi.vsphere.vmware.com`?

Reply:
will take up in separate PR